pyproject.toml

```diff
@@ -4,10 +4,10 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "basyx-python-sdk"
-version = "0.1.0"  # Replace with your actual version
+version = "1.0.0"  # actual version
 description = "The Eclipse BaSyx Python SDK, an implementation of the Asset Administration Shell for Industry 4.0 systems"
 authors = [
-    {name = "The Eclipse BaSyx Authors", email = "example@example.com"}  # Replace with a real email
+    {name = "The Eclipse BaSyx Authors", email = "h.jannatabadi@iat.rwth-aachen.de"}  # actual authors
```

Comment:
If you cannot leave out the E-Mail field, I'd use `admins@iat.rwth-aachen.de`.
Same thing as above about the comment.